extensions/shadcn-ui/src/search-components.tsx

```diff
@@ -0,0 +1,88 @@
+import { ActionPanel, Action, List, showToast, Toast } from "@raycast/api";
+import { useFetch, Response } from "@raycast/utils";
+
+export default function Command() {
+  const { data, isLoading } = useFetch("https://ui.shadcn.com/api/components", {
+    parseResponse: parseFetchResponse,
+  });
+
+  return (
+    <List isLoading={isLoading} searchBarPlaceholder="Search components...">
+      <List.Section title="Components">
```

Comment:
in lists that don't have separations, maybe a hardcoded section doesn't add much value. Just my opinion

Reply:
Nice one, I agree

done fc917ed98aec3370adb14eb9716bab9dabc3f1af